requirements.txt

```diff
@@ -1,4 +1,4 @@
-scikit-learn~=0.18
+scikit-learn>=0.18.1
```

Comment:
We should prob use this

```
scikit-learn~=0.18,>=0.18.1
```

so that we don't bump a version.

Reply:
OTOH, we could just require sklearn 0.19.

Reply:
Hmmm. Would `scikit-learn~=0.18,>=0.18.1` prohibit 0.19? I'd like to be as permissive as possible, and things seem to work with any release >= 0.18.1 (including 0.19). What do you think?

Reply:
Maybe `scikit-learn>=0.18.1,<0.20`?